sdk/appconfiguration/azure-appconfiguration-provider/README.md

```diff
@@ -60,8 +60,7 @@ You can refine or expand the configurations loaded from your store by using `Set
 
 ```python
 selects = {SettingSelector(key_filter="*", label_filter="\0"), SettingSelector(key_filter="*", label_filter="dev")}
-config = AzureAppConfigurationProvider.load(
-    endpoint=endpoint, credential=default_credential, selects=selects)
+config = load_provider(endpoint=endpoint, credential=default_credential, selects=selects)
```

Comment:
Where is default_credential defined?

Reply:
No where here, I think the README assumes that they know how to make one. Should I add it?

Reply:
We received some feedback reporting that it was confusing with such stuff. :)

Maybe you can just use DefaultAzureCredential().